CHANGELOG.md

```diff
@@ -20,8 +20,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * [Polymer] Extract pseudo elements from HTML comments
 
 ### Fixed
-* Fixed a class of race conditions and cache invalidation errors that can occur when there are concurrent analysis runs and edits to files.
-
+* Fixed a class of race conditions and cache invalidation errors that can occur when there are parallel analysis runs and edits to files.
```

Comment:
Arg, you're right, this gets me every time. The technical definitions of these words just seem so intuitively backwards to me.

`<rant please-ignore>`

In the nontechnical dictionary sense, concurrent means multiple things happening simultaneously, and parallel means two things being side by side. When there are two threads of execution they're naturally represented as parallel lines, and even when they run on one core they're still separate linear sequences of operations that just contend for the core.

`</rant>`